src/app/sign in/page.jsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import Link from "next/link";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+import React from "react";
+
+export default function Login() {
+  return (
+    <center>
+      <main className="flex min-h-screen flex-col items-center p-24">
+        <Box
+          component="section"
+          sx={{ borderRadius: 1, p: 5, border: "1px solid grey" }}
+        >
+          <div className="blank"></div>
```

Comment:
Instead of having 3 divs with with className "blank" before and after the h1, can you remove them and apply the padding on that h1 tag